<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user</class>
 <widget class="QWidget" name="user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>680</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 626.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>-10</y>
     <width>1061</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 36, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>1061</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>1031</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/playlist2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>1031</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/Frame 604.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playPauseButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/play.png);

    background-color: transparent;
    border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>11</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="songTableWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>1031</width>
     <height>341</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="urutkanButton">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>320</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Urutkan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hapusButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>320</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2A2A2A;
color: white;
padding: 6px;
border: none;
border-radius: 6px;
</string>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="progressSlider">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="currentTimeLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>backButton</zorder>
  <zorder>playPauseButton</zorder>
  <zorder>songTableWidget</zorder>
  <zorder>urutkanButton</zorder>
  <zorder>hapusButton</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>progressSlider</zorder>
  <zorder>currentTimeLabel</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
